doc/devguide.md

```diff
@@ -0,0 +1,130 @@
+# Azure IoT SDK Developer Documentation
+
+## 1. Dev machine prerequisites
+
+Before starting development on the Azure IoT SDK for C# you will need to install a few frameworks and tools. Please follow the instructions within [devbox_setup](devbox_setup.md).
+
+## 2. Coding style
+
+Please read and apply our [coding style](coding-style.md) when proposing PRs against the Azure repository.
+
+## 3. Cloning the repository
+
+`git clone https://github.com/Azure/azure-iot-sdk-csharp.git`
```

Comment:
>` [](start = 60, length = 1)

Error with the link. Remove "`"

Reply:
That's MD verbatim formatting. This is not a link but a copy-paste-able command. Looks good when you view the end result: https://github.com/abhipsaMisra/azure-iot-sdk-csharp/blob/554b2ccdc189f1d616ab47639f4dd7f67a7934f3/doc/devguide.md